textproc/marisa-trie/Portfile

```diff
@@ -28,8 +28,9 @@ long_description    Matching Algorithm with Recursively Implemented StorAge \
 worksrcdir          ${worksrcdir}
 distname            v${version}
 
-test.run            yes
-test.target         test
+cmake.build_type    Release
+
+configure.args      -DENABLE_NATIVE_CODE=ON
```

Comment:
This should not be enabled, since it will add non-portable cpu-specific optimizations. End-users may not have identical cpus to MacPorts buildbots. (It will also break powerpc builds, since the flag is wrong.)